Makefile

```diff
@@ -7,18 +7,16 @@ define PROJECT_ENV
 []
 endef
 
-# FIXME: Use erlang.mk patched for RabbitMQ, while waiting for PRs to be
-# reviewed and merged.
-
-ERLANG_MK_REPO = https://github.com/rabbitmq/erlang.mk.git
-ERLANG_MK_COMMIT = rabbitmq-tmp
-
 LOCAL_DEPS = crypto inets ssl xmerl
 BUILD_DEPS = rabbit_common
 DEP_PLUGINS = rabbit_common/mk/rabbitmq-plugin.mk
 TEST_DEPS = meck
 
-current_rmq_ref = master
+# FIXME: Use erlang.mk patched for RabbitMQ, while waiting for PRs to be
+# reviewed and merged.
+
+ERLANG_MK_REPO = https://github.com/rabbitmq/erlang.mk.git
+ERLANG_MK_COMMIT = rabbitmq-tmp
 
 include rabbitmq-components.mk
-include $(if $(ERLANG_MK_FILENAME),$(ERLANG_MK_FILENAME),erlang.mk)
```

Comment:
Was this an intentional change?

Reply:
Yep, I brought this `Makefile` more in-line with the other RabbitMQ modules. I used `rabbitmq_peer_discovery_aws` as the model. `rabbitmq_aws` was the outlier. The `ERLANG_MK_FILENAME` variable is not used.